webpack.common.config.ts

```diff
@@ -617,7 +619,16 @@ module.exports = {
     }),
     new ForkTsCheckerWebpackPlugin({
       checkSyntacticErrors: true
-    })
+    }),
+    new webpack.LoaderOptionsPlugin({
+      options: {
+        macros: {
+          load: macros.load,
+          loadExtensions: macros.loadExtensions,
+          loadHeader: macros.loadHeader
```

Comment:
`loadHeader` is not used in the HTML files and also not defined.

Reply:
Removed.
Thanks.